src/pdn/test/pads_black_parrot_flipchip_connect_overpads.defok

```diff
@@ -4110,13 +4110,12 @@ SPECIALNETS 139 ;
       NEW metal9 2800 + SHAPE STRIPE ( 381140 1765800 ) ( 5618680 1765800 )
       NEW metal9 2800 + SHAPE STRIPE ( 381140 1685800 ) ( 5902000 1685800 )
       NEW metal9 2800 + SHAPE STRIPE ( 381140 1445800 ) ( 5618680 1445800 )
-      NEW metal9 2800 + SHAPE STRIPE ( 381140 1365800 ) ( 5902000 1365800 )
+      NEW metal9 2800 + SHAPE STRIPE ( 381140 1365800 ) ( 5632680 1365800 )
```

Comment:
Does this test have negative halos?

Reply:
No, but it has a strap that was very close to the bump shape (which now gets an obstruction box added around it), so that shape gets cut a little shorter